Functions.Templates/Templates/FaceLocator-CSharp/metadata.json

```diff
@@ -1,12 +1,13 @@
 {
     "defaultFunctionName": "FaceLocatorCSharp",
```

Comment:
Not a necessity, but it bothers me we're not ordering these properties in any enforced way

Reply:
Ya that was bugging me too. Made it hard to do these updates as well. I'm changing them now so the first 3 are consistent in their order (defaultFunctionName, description, name)